Expensify/App
# #2806 fix selectable timezone dropdown with thuthy disabled passed

### Details
### Fixed Issues
Fixes #2806

### Tests / QA Steps

1. Go to Setting - Profile
2. Scroll down to Time zone field
3. Set my time zone automatically should be selected
4. The time zone picker should be disabled (clicking on it must not bring up any options)

### Tested On


### Screenshots


#### Web
<img width="500" alt="DeepinScreenshot_select-area_20210513132039" src="https://user-images.githubusercontent.com/64093836/118112992-64395780-b3ee-11eb-93ee-19fee6073719.png">


#### Mobile Web

<img width="250" alt="DeepinScreenshot_select-area_20210513132049" src="https://user-images.githubusercontent.com/64093836/118113018-6b606580-b3ee-11eb-98b0-09b38c5c3126.png">



#### Desktop

<img width="500" alt="Screen Shot 2021-05-13 at 13 25 58" src="https://user-images.githubusercontent.com/64093836/118113361-dd38af00-b3ee-11eb-9c9c-37122fb5d022.png">


#### iOS

<img width="250" alt="Screen Shot 2021-05-13 at 13 25 06" src="https://user-images.githubusercontent.com/64093836/118113343-d611a100-b3ee-11eb-8ef0-ede776356959.png">


#### Android

<img width="250" alt="DeepinScreenshot_select-area_20210513132012" src="https://user-images.githubusercontent.com/64093836/118113046-73200a00-b3ee-11eb-9a52-b28a381a5380.png">
